src/python/pants/engine/target.py

```diff
@@ -702,9 +702,13 @@ class Dependencies(AsyncField):
     sanitized_raw_value: Optional[Tuple[Address, ...]]
     default = None
 
+    # NB: The type hint for `raw_value` is sort of a lie. While we do expect end-users to use
+    # Iterable[str], the Struct and Addressable code will have already converted those strings
+    # into a List[Address]. But, that's an implementation detail and we don't want our
```

Comment:
Might be worth explaining why the documentation would be affected (i.e., that it's generated from the type annotations).